modules/core/include/opencv2/core/softfloat.hpp

```diff
@@ -389,6 +389,9 @@ CV_EXPORTS int cvTrunc(const cv::softdouble& a);
 CV_EXPORTS int cvRound(const cv::softfloat&  a);
 CV_EXPORTS int cvRound(const cv::softdouble& a);
 
+/** @brief Rounds a number to nearest even long long integer */
+CV_EXPORTS int64_t cvRound64(const cv::softdouble& a);
```

Comment:
Perhaps we need test for this here: https://github.com/opencv/opencv/blob/master/modules/core/test/test_math.cpp